scss/layout/_grid.scss

```diff
@@ -48,6 +50,21 @@
 
 // BREAKPOINT: LARGE
 @include media($lg) {
+  .grid--3-wide {
+    .grid__item {
+      @include span-columns(4);
+      margin-bottom: 0;
```

Comment:
Not sure. This is currently used in places where we have a 2x2 or 4x1 grid. In the case of the 2x2, you want to have the margin-bottom set at all sizes; and in the case of 4x1, you want it just set at small screens when it becomes a 1x4. 

Not sure what the right way to abstract is. Suggestions?